Added calibration 14 data struct

valegagge opened the pull request:
In this PR we added the data related to calibration 14: it is used for the open close joint f ergoCub fingers.

### **Important note**
Even if the calibration 14 parameters hosts both motor encoder info and joint encoder (`FAP`)info,  currently only the data regarding the motor encoder are used: `pwmlimit` and `final_pos.` 

----------------

### Related PRs:
 - [fw PR](https://github.com/robotology/icub-firmware/pull/363)
 - [icub-main PR](https://github.com/robotology/icub-main/pull/862)




cc @MSECode

pattacini commented:
Hi @valegagge 

Probably, we should handle the package versioning.
Consequently, `icub-main@devel` should be upgraded to link to the latest version.

cc @valegagge

marcoaccame commented:
linked to:
- https://github.com/robotology/icub-firmware/pull/363

it probably needs a PR also on https://github.com/robotology/icub-firmware-build

marcoaccame commented:
> Hi @valegagge
> 
> Probably, we should handle the package versioning. Consequently, `icub-main@devel` should be upgraded to link to the latest version.
> 
> cc @valegagge

yes

valegagge commented:
Oh yes for sure!! I'll add it asap!!!

valegagge commented:
> linked to:
> - https://github.com/robotology/icub-firmware/pull/363
> 
> it probably needs a PR also on https://github.com/robotology/icub-firmware-build
> 

Yes, we are doing it

valegagge commented:
I updated the version of `icub-firmware-shared` to `[1.34.1](https://github.com/robotology/icub-firmware-shared/pull/81/commits/6eb74e9b020b9690904960113c59599dd8010889)` and updated according `icub-main`

marcoaccame reviewed (approved):
It is OK by me.
**However**, we shall merge it only when the other PRs are OK. See:
- https://github.com/robotology/icub-firmware/pull/363
- https://github.com/robotology/robots-configuration/pull/472
- TBD on binaries
